### PR TITLE
More enum abstracts

### PR DESCRIPTION
**Pull request by RblSb**
Functions with int switches removed. `@:expose` from `kha.Color` too.

### CODE REVIEW

File: Sources/kha/compute/Access.hx [hunk 1]
```diff
@@ -1,7 +1,11 @@
 package kha.compute;
 
-enum Access {
-	Read;
-	Write;
-	ReadWrite;
+enum abstract Access(Int) {
```
Comment (sh-dave) [inline Sources/kha/compute/Access.hx]: just add `to Int` and remove the toInt() function (if you need the int conversion at all)?

Comment (RblSb) [reply Sources/kha/compute/Access.hx]: I think explicit `to` functions is little better for such enums (when you need to get int only for internal apis), but `to Int` is fine too

Comment (sh-dave) [reply Sources/kha/compute/Access.hx]: just wondering, why do you think it's better?

Comment (RblSb) [reply Sources/kha/compute/Access.hx]: You can send that abstract as int argument accidentally and switch only for `Read` and `Write` without errors, but this is very minor =)

**Pull request merged**